test/unit/BPool.t.sol

```diff
@@ -818,195 +818,6 @@ contract BPool_Unit_GetSpotPriceSansFee is BasePoolTest {
   }
 }
 
-contract BPool_Unit_ExitPool is BasePoolTest {
```

Comment:
i think this is not the test you should be erasing in this PR

Reply:
:fearful:

Reply:
ahh perhaps was on the "merging dev" commit